docs/book/src/tasks/automated-machine-management/healthchecking.md

```diff
@@ -92,6 +92,50 @@ in order to prevent conflicts or unexpected behaviors when trying to remediate t
 
 </aside>
 
+## Controlling remediation retries
+
+<aside class="note warning">
+
+<h1> Important </h1>
+
+This feature is only available for KubeadmControlPlane.
+
+</aside>
+
+KubeadmControlPlane allows to control how remediation happen by defining an optional `remediationStrategy`;
+this feature can be used for preventing unnecessary load on infrastructure provider e.g. in case of quota problems,
+or for allowing the infrastructure provider to stabilize in case of temporary problems:
```

Comment:
```suggestion
KubeadmControlPlane enables control of machine remediation by defining an optional `remediationStrategy`;
this feature can be used for preventing unnecessary load on infrastructure provider e.g. in case of quota problems,or for allowing the infrastructure provider to stabilize in case of temporary problems.
```